extensions/reference/extension-manifest.md

```diff
@@ -345,12 +374,14 @@ When the settings object is emitted in the Launch runtime library, it will be tr
 
 ```javascript
 {
-  foo: {
-  }
+  foo: [
+    {},
+    {}
+  ]
```

Comment:
I think showing an array example for the remove transform is noble, but may muddy communication of how the remove transform would be used in 95+% of the cases. Even for the more advanced extension developer dealing with arrays of objects, it may come off a bit confusing as to why we removed `foo[].bar` instead of just removing `foo` entirely, even though you and I know it's a somewhat contrived example to demonstrate how array handling works. I'd probably just leave the remove transform example the way it was, but feel free to do what you think is best here.

Reply:
Yeah that makes sense to me. I wanted to take a stab to see how it felt but I was leaning towards removal as well.